Mergin/plugin.py

```diff
@@ -108,6 +108,10 @@ def download(self):
             msg = "Failed to download your project {}.\n" \
                   "Please make sure your Mergin settings are correct".format(self.project_name)
             QMessageBox.critical(None, 'Project download', msg, QMessageBox.Close)
+        except LoginError as e:
+            QgsApplication.messageLog().logMessage(f"Mergin plugin: {str(e)}")
+            msg = f"<font color=red>Security token has been expired, try to renew failed, {str(e)} </font>"
```

Comment:
is not LoginError raised only if automatic token renewal failed? That would imply that probably username/password is incorrect or missing. So I would inform user about that (e.g. check configuration)